src/pt/sussyscan/src/eu/kanade/tachiyomi/extension/pt/sussyscan/SussyToons.kt

```diff
@@ -316,13 +316,15 @@ class SussyToons : HttpSource(), ConfigurableSource {
             .filter(String::isNotEmpty)
             .joinToString("\n")
 
-        val content = quickJs.evaluate(
-            """
+        val content = quickJs.use {
```

Comment:
Don't need the variable elsewhere
```suggestion
        val content = QuickJs.create().use {
```